src/auth/auth-config.ts

```diff
@@ -506,6 +507,27 @@ export interface MultiFactorConfig {
    * Currently only ‘phone’ is supported.
    */
   factorIds?: AuthFactorType[];
+  /**
+   * A list of multi-factor provider specific config.
```

Comment:
Extend the comment with "new MFA providers (except phone) will indicate enablement/disablement through this field."